intermediate_html_css/intermediate_css_concepts/default_styles.md

```diff
@@ -11,34 +11,38 @@ This section contains a general overview of topics that you will learn in this l
 
-To counter this, many developers start their projects with a "CSS Reset"; a file that undoes browser defaults, so that every element behaves the same in every browser.
-
-You don't _have_ to use a CSS reset. In many cases, you're going to end up undoing, or redoing a lot of the styles that a reset will provide for you. You can decide whether or not you want to use one on a regular basis, but it's worth taking the time to dig through a couple now. Understanding exactly how they're doing what they're doing is a useful exercise!
+<a id="resets-optional"></a>While CSS resets are still commonly used, they are not mandatory. Some developers opt not to use them, while others create their own or utilize prebuilt resets. It's important to understand that resets are subjective and opinionated, reflecting the preferences of the developer who created them. You can decide how you wish to do things yourself.
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
 
-1.  [The Meyer Reset](https://meyerweb.com/eric/tools/css/reset/) is almost certainly the most popular. It basically removes every default style.
-2.  [Normalize.css](http://nicolasgallagher.com/about-normalize-css/) is another popular one. It's a little different in that it doesn't remove all the default styles, but tweaks them slightly to ensure that browsers are consistent. Currently, the article's download link is dead. If you want to install normalize.css for your project, download it from [normalize.css official website](https://necolas.github.io/normalize.css/) or use the instructions found in the [normalize.css repository's](https://github.com/necolas/normalize.css) `README.md` file. If you are unsure which method to use, go with the NPM package install.
-3.  [Reboot, Resets and Reasoning](https://css-tricks.com/reboot-resets-reasoning/) is a CSS tricks article that goes a little more in-depth, and mentions a few other popular resets.
-4.  Maybe more trivial than useful, this [Browser Default Styles](https://browserdefaultstyles.com/) site is fun to play with.
+1. Read [Reboot, Resets, and Reasoning](https://css-tricks.com/reboot-resets-reasoning/) for an excellent history of resets and what it means for a reset to be opinionated.
+1. Read [Making the case for CSS normalize and reset stylesheets in 2023](https://mattbrictson.com/blog/css-normalize-and-reset). It does an excellent job of discussing the differences in various resets and why you might choose to use them.
+1. Elad Schecter takes a fresh approach to using resets in [The New CSS Reset](https://elad.medium.com/the-new-css-reset-53f41f13282e), including the use of a reset and normalization. The reset illustrates how resets change as browsers implement new CSS tools.
+1. [Josh Comeau's custom CSS reset](https://www.joshwcomeau.com/css/custom-css-reset/) also includes a great breakdown of his thought process behind each rule he uses, giving you an idea of how to reason about these ideas.
```

Comment:
Makes sense to me, but I am conflicted as to which one to remove.  I am leaning towards Elad's as it is a bit advanced, but I like that it is such a dramatically different approach.  A second for me woudl be Coumeau's, but I love his stuff, so I just cant decide...

I could use some opions on this....

Reply:
We can make @MaoShizhong decide 😈 

My first thought was Elad's as well for the same reasons you stated, but honestly it might be between that and the "Making the case for CSS normalize and reset stylesheets in 2023" one if we wanted to cut at least one.

Again, I'd also be good with shipping all of them and let feedback help us determine if we need to cut some stuff out as a followup.

Reply:
Let me read between the Elad and CSS Normalize ones and see how I feel whether we should definitely keep both or definitely axe one.

Reply:
A more thorough read-through of all of them, I'm of the opinion that we can drop the Elad article.
They're all great reads, don't get me wrong, and the Elad one definitely has some benefits from going deeper with a much more different approach from some of the others, but in the context of someone at this point in the curriculum, I don't feel like that more advanced and different angle will be as beneficial/easy to appreciate as it may be to people like us with a little more practical understand of this all.

I think the other 3 articles are sufficient to get the main points across while reining in the scope to a more sensible radius. How does that sound?

Reply:
Ok, done.